### ISSUE TITLE
Windows installer doesn't remove zope.interface pyc at uninstall

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3164]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#3164|
|Type|defect|
|Created|2008-04-04 20:34:50Z|

It should.

Attachments:

* [3164.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/cbf/cbfe42b5899c620643436dbbed6ee9bb2decde69/0d74e27efc355ae595391d11da51b1f3bbc33674.diff) (489 bytes) - added by therve on 2008-04-04 20:35:21Z - 
* [3164.2.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/cbf/cbfe42b5899c620643436dbbed6ee9bb2decde69/a9589f6c245c83960defb3c2299cc3cd601013be.diff) (631 bytes) - added by therve on 2008-04-07 18:37:49Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3164 3164
type__defect defect
reporter__therve therve
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1207341290000000 1207341290000000
changetime__1207726590000000 1207726590000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix set owner to @therve|
|-|-|

Is this really sufficient? I wonder if there would be a zope/__init__.pyc for example.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve removed owner|
|-|-|

You were right. It doesn't happen with setuptools, but without it does. Also you can have some pyo lying around.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set owner to @therve|
|-|-|

+1 THANKS!

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set status to `closed`|
|-|-|

(In [23264]) Fix windows installer: remove pyc and pyo files of zope.interface package at
uninstall so that the directories are correctly removed.

Author: therve
Reviewer: radix
Fixes [#3164](https://github.com/twisted/twisted/issues/3164)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
